.gitignore

```diff
@@ -21,4 +21,5 @@ metastore_db/
 derby.log
 dependency-reduced-pom.xml
 rss-*.tgz
-
+hadoop-*.tar.gz
+/deploy/kubernetes/hadoopconfig/*
```

Comment:
Why do we use absolute path here? if `hadoopconfig` is useful, why do we add it to `.gitignore` file?